src/l500/l500-options.cpp

```diff
@@ -36,23 +36,52 @@ namespace librealsense
         return _range;
     }
 
+    float l500_hw_options::query_default() const
+    {
+        if(_fw_version >= firmware_version( "1.5.3.0" ) )
+        {
+            return query( get_default, int( _resolution->query() ) );
+        }
+        else
+        {
+            // FW versions older than get_default doesn't support
+            // the way to get the default is set -1 and query
+            auto current = query( get_current, int( _resolution->query() ) );
+            _hw_monitor->send( command{ AMCSET, _type, -1 } );
+
+            // if the sensor is streaming the value of control will update only whan the next frame
+            // arrive
+            if( _l500_dev->get_depth_sensor().is_streaming() )
+                std::this_thread::sleep_for( std::chrono::seconds( 50 ) );
+
+            auto def = query( get_current, int( _resolution->query() ) );
```

Comment:
This works well? Have you tried it?
One optimization I can think of is to set all the controls to -1, wait 50ms, then query the value of all the controls -- so we wait 50ms one time.